meta/main.yml

```diff
@@ -1,27 +1,57 @@
----
 galaxy_info:
-  author: Werner Dijkerman
-  description: Installing and maintaining the ossec-agent.
-  company: myCompany.Dotcom
-  license: MIT
-  min_ansible_version: 2.4
-  platforms:
-    - name: EL
-      versions:
-        - 5
-        - 6
-        - 7
-    - name: Fedora
-      versions:
-        - all
-    - name: Debian
-      versions:
-        - all
-  categories:
-    - monitoring
-  galaxy_tags:
-    - ossec
-    - ossec-agent
-    - hids
-    - monitoring
+  author: your name
```

Comment:
This too, is default molecule stuff

Reply:
Also addressed in c64e407b40169f724ea038614fe77dad27c4b662 to have correct information